rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/jobs/local/JobExecutorLocal.groovy

```diff
@@ -110,6 +110,22 @@ class JobExecutorLocal implements JobExecutor {
   }
 
   @Override
+  BakeStatus getJob(String jobId) {
+    try {
+      if (jobIdToHandlerMap[jobId]) {
+        return new BakeStatus(id: jobId, resource_id: jobId)
+      } else {
+        // This instance of rosco is not managing the job.
+        return null;
+      }
+    } catch (Exception e) {
```

Comment:
Please drop this exception clause entirely, unless there is some expected exception I'm missing. If there is, we can use that specific exception type.